front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -125,22 +121,15 @@ const IdeasShowPage = () => {
             compact={isSmallerThanTablet}
           />
         </Box>
-        <Box
-          position="fixed"
-          top={
-            isSmallerThanTablet
-              ? undefined
-              : stylingConsts.menuHeight.toString() // Show CTA at top of screen on desktop
-          }
-          bottom={isSmallerThanTablet ? '0px' : undefined} // Show CTA at bottom of screen on mobile
-          width="100vw"
-          zIndex="1000"
-        >
-          {project?.data.id && // Show the CTA bar only when voting is active
-            isIdeaInCurrentPhase &&
-            participationContext?.attributes.participation_method ===
-              'voting' && <ProjectCTABar projectId={project.data.id} />}
-        </Box>
```

Comment:
- Always try to wrap the outermost component with the condition to not render unnecessary code/unexpected empty spaces in the UI.
- I didn't see any changes to the UI after removing zIndex and top, so I removed them. Was zIndex added for a reason? Was something overlapping it?

Reply:
@brentguf Yeah you're right - they can be removed :thinking: I think I add things sometimes which are required for whatever initial approach I have, but then later on they become unnecessary. I'll try to catch them myself next time :) Thanks again for the thorough review!